api/db-cdr/generate-cdr/cb-criteria-refactor/make-cb-criteria-21-seq-03-clean-up-text-synonym.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
```

Comment:
Could we merge missing-codes, attrib-other-tables and clean-up-text-synonyms into the same script since they are run sequentially according to you workflow diagram?

Reply:
These files are ~150-400 lines each, so it maybe make sense to keep these separate.